packages/react-devtools-extensions/deploy.js

```diff
@@ -18,11 +18,9 @@ const main = async buildId => {
     },
     stdio: 'inherit',
   });
-
-  await exec(`cp ${join(root, 'now.json')} ${join(buildPath, 'now.json')}`, {
-    cwd: root,
-  });
-
+  shell.pushd(root);
+  shell.cp(join(root, 'now.json'), join(buildPath, 'now.json'));
+  shell.popd();
```

Comment:
I think both `root` and `buildPath` should be absolute paths. If so, then specifying `cwd` is not required, so we can remove `pushd` and `popd` calls

Reply:
Sounds good! I was wondering the same thing